core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -780,6 +780,22 @@ public List<LogRecord> getLogRecords() throws IOException, InterruptedException
             return channel.call(new SlaveLogFetcher());
     }
 
+    /**
+     * Inline editing of description
+     */
```

Comment:
```suggestion
    /**
     * Inline editing of description
     * @since TODO
     */
```

Reply:
It's not part of the public Java API so it shouldn't have `@since` could be restricted though

Reply:
If it's not intended to be public API, we should restrict it then 👍🏻